public/assets/js/scripts.js

```diff
@@ -285,4 +286,49 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    const periodic = $('.periodic-investment').val();
+    const initial = $('.initial-investment').val();
+    const avgPropertyCost = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
+    const frequency = $("input[name='investment-frequency']:checked").val();
+
+    let balance = avgPropertyCost - initial;
+    let output;
+    let paymentType;
+    
+    let outrightPersonal = initial >= (avgPropertyCost * 0.5) && balance / periodic < 6 / frequency;
```

Comment:
Please leave a comment linking to the document where the category breakdown is made to serve as a reference to anyone reviewing/updating this code later.